ctapipe/core/traits.py

```diff
@@ -138,9 +138,20 @@ def has_traits(cls, ignore=("config", "parent")):
     return bool(set(cls.class_trait_names()) - set(ignore))
 
 
-class TelescopeParameterList(list):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+class TelescopeParameterLookup:
+    def __init__(self, telecope_parameter_list=None):
+        """
+        Handles the lookup of corresponding configuration value from a list of
+        tuples for a telid.
+
+        Parameters
+        ----------
+        telecope_parameter_list : list
+            List of tuples in the form `[(command, argument, value), ...]`
+        """
+        if telecope_parameter_list is None:
+            telecope_parameter_list = []
```

Comment:
Why passing `None` as the default value for `telescope_parameter_list` and not directly `[ ]`?

Reply:
Because a list is mutable:
https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments